### ISSUE TITLE
WOW not defined using npm package.

### ISSUE THREAD

**Issue by jasonlam0619** [matthieua/WOW#252]
I did this:

`npm install wowjs`

and required it:

`import WOW from 'wowjs`

and tried to use it:

    let wow = new WOW({
      boxClass: 'wow',
      animateClass: 'animated',
      offset: 0,
      live: true
    });

But I got `Uncaught ReferenceError: WOW is not defined` at console.

**Comment by graingert** [comment]
@jasonlam0619 run:
```
npm install wow.js
```
```js
import WOW from 'wowjs'
const wow = new WOW({
  boxClass: 'wow',
  animateClass: 'animated',
  offset: 0,
  live: true
});
```

**Comment by aaesis** [comment]
Hi grain I used wow.js instead of wowjs, because it is not working with import ... from 'wowjs'

and then tried the code you have provided.

when I run in my html
`<h1 class="nomargin size-50 weight-300 wow rollIn" data-wow-delay="0.5s">
                blah blah
            </h1>
            <p class="lead font-lato size-30 wow fadeInUp" data-wow-delay="0.7s">
               blah blah
            </p>`

the output is it has no animate, see the screenshot:

<img width="790" alt="screen shot 2016-09-06 at 11 41 22 pm" src="https://cloud.githubusercontent.com/assets/802180/18280502/ca5b2b08-748b-11e6-9681-74682cf5f7d6.png">

**Comment by VanDavv** [comment]
I got  this error when using @graingert example

> myWow.js:11 Uncaught TypeError: _wowjs2.default is not a constructor

**Comment by graphan** [comment]
@graingert unfortunately, the code you mentioned does not work. Something changed?

**Comment by kaueb** [comment]
@graingert i have tried this code, but did not worked :/

**Comment by jsantana90** [comment]
any luck here?

**Comment by kaueburiti** [comment]
@jsantana90 i totally give up

**Comment by suncoastkid** [comment]
Same issue here...

**Comment by shealan** [comment]
Same issue. Surely rather important as most developers are using NPM dependencies these days?

**Comment by badalsurana** [comment]
same issue here, anyone found any solution?

**Comment by interphased** [comment]
Not sure if this helps, but I got it to work using `require` and adding it to `window`.

_bootstrap.js_
```
const WOW = require('wowjs');

window.wow = new WOW.WOW({
    live: false
});
```

_whatever.js_
```
window.wow.init();
```

**Comment by ruchern** [comment]
`new WOW.WOW().init();`

in your HTML file works instead of having to place it in your `app.js` to compile.

**Comment by adammoisa** [comment]
This didn't work either. I used `new WOW.WOW().init()` but it still doesn't work. I can see in the rendered html that the animated class is indeed being added, but the animation doesn't run.

**Comment by ruchern** [comment]
@adammoisa Did you import WOW in anywhere of your scripts?

**Comment by Steven-Garcia** [comment]
I'm getting the same problem, it used to work fine with this line in the controller: new WOW().init(); and importing the script in the index.html, now I get the same error and it just won't work at all

**Comment by adammoisa** [comment]
@Ruchern yes of course. As I said it adds the animated class to the HTML which I think implies the package is 'working' to some degree...

**Comment by ruchern** [comment]
@adammoisa I'm sorry. I did not mean that.

Do you have `window.WOW = require('wow-js')`?

**Comment by gaboesquivel** [comment]
the module exports an object, the WOW constructor function is inside the returned object.

`const WOW = require('wowjs').WOW`

**Comment by xHeinrich** [comment]
window.WOW = require('wowjs').WOW worked for me

**Comment by ruchern** [comment]
``` javascript
import { WOW } from 'wowjs'
window.WOW = WOW
```

Perhaps this might work too.

**Comment by Sydney-o9** [comment]
# With require

```
const {WOW} = require('wowjs');
```

**Comment by TanvirAmi** [comment]
To Leverage Undefined Values make sure that your DOMm is ready and likely when a dependent variable is not defined. When the page is loaded and ready, and likely when a dependent variable is already defined.
For this use `$(document).ready(function() {
    if ($.fn.init) {
        new WOW().init();
    }
}`

**Comment by jhoanborges** [comment]
> window.WOW = require('wowjs').WOW worked for me

This works like a charm

**Comment by BenjaminHoppe** [comment]
> window.WOW = require('wowjs').WOW worked for me

hmmm, now I'm getting the error 'require not defined'

**Comment by mugianto** [comment]
> 
> 
> window.WOW = require('wowjs').WOW worked for me

Thanks bro this work for me

**Comment by 766** [comment]
> Not sure if this helps, but I got it to work using `require` and adding it to `window`.
> 
> _bootstrap.js_
> 
> ```
> const WOW = require('wowjs');
> 
> window.wow = new WOW.WOW({
>     live: false
> });
> ```
> 
> _whatever.js_
> 
> ```
> window.wow.init();
> ```

Thanks bro this work for me

**Comment by vipertecpro** [comment]
```
window.WOW = require('wowjs').WOW;

  function wowAnimation() {
    new WOW({
      boxClass:   'wow',
      animateClass: 'animated',
      offset: 0,
      mobile: true
    }).init();
  }

  wowAnimation();
```

**Comment by kshvab** [comment]
React + Next js + typescript Works fine:

```
useEffect(() => {
        const WOW = require("wowjs/dist/wow.js");
        if (typeof window !== "undefined") {
            (window as any).WOW = WOW;
        }

        new WOW.WOW().init();
    }, []);
```

**Comment by veraLX** [comment]
Hello，我是林昕，我已收到来信~

**Comment by Aaryan6** [comment]
Try this, It works for me

// import wowjs if window is not undefined
```
const isServer = typeof window === "undefined";
const WOW = !isServer ? require("wowjs") : null;
```

// then write this code in the function
```
useEffect(() => {
    new WOW.WOW().init();
  }, []);
```

I hope this is useful ; )

**Comment by harapeko** [comment]
it works for me : )

- "@sveltejs/kit": "1.15.2",
- "wow.js": "^1.2.2" (not wowjs)

```sh
npm i --save wow.js
```

```ts
import { onMount } from 'svelte'

onMount(async () => {
	import('wow.js').then((WOW) => {
		new WOW.default().init()
	})
})
```

or 

```ts
onMount(async () => {
	const wow = await import('wow.js')
	new wow.default().init()
})
```

**Comment by Cayllen** [comment]
https://github.com/matthieua/WOW/issues/252#issuecomment-1517949732
 Harepo how does it work for you with sveltekit? I have it implemented but none of the animations work. Did you do have to do some extra work to make it work?

**Comment by nelson-k-808** [comment]
I found a better way to do it and it works.

Currently I’m using 

* “vue” : “3.3.4”
* “wow.js” : ”1.2.2”
* “ animate.css”: “4.1.1”

1. Install `wow.js` 

       npm install wow.js

2. In your SFC or *.vue file add this

       onMounted(async () => {
         const wow = await import(‘wow.js’)
         new wow.default().init()
       }
           
3.  In your template add the **wow** class to any     HTML tag element like this 
      
     ```
     <template>
       <div class=“wow fadeInUp”></div>
     </template>
      ```

**Comment by veraLX** [comment]
Hello，我是林昕，我已收到来信~
